datastore.go

```diff
@@ -374,3 +382,31 @@ func (api *API) PasswordDatastore(ctx context.Context, datastoreID string, opts
 
 	return result.Datastore, nil
 }
+
+func (api *API) BackupsDatastore(ctx context.Context, datastoreID string, opts DatastoreBackupsOpts) (Datastore, error) { //nolint
```

Comment:
You should add a docstring for this method

Reply:
Also, you need to add tests for a new endpoint to the datastore_test.go

Reply:
Also, it would be great if you add some tests for the function to the `datastore_test.go`

Reply:
Done